src/features/Settings/Settings.tsx

```diff
@@ -49,6 +55,65 @@ const StyledLayout = styled.div`
   }
 `;
 
+function rendedExcludedAssetsPanel() {
+  const { accounts, totals, currentAccounts } = useContext(StoreContext);
+  const { settings, removeAssetfromExclusionList } = useContext(SettingsContext);
+  const { getAssetRate } = useContext(RatesContext);
+
+  const balances: Balance[] = totals(currentAccounts)
+    .filter((asset: StoreAsset) => {
+      return !isExcludedAsset(settings.excludedAssets)(asset);
+    })
+    .map((asset: StoreAsset) => {
+      const exchangeRate = getAssetRate(asset);
+      return {
+        id: `${asset.name}-${asset.ticker}`,
+        name: asset.name || translateRaw('WALLET_BREAKDOWN_UNKNOWN'),
+        ticker: asset.ticker,
+        uuid: asset.uuid,
+        amount: weiToFloat(asset.balance, asset.decimal),
+        fiatValue: convertToFiatFromAsset(asset, exchangeRate),
+        exchangeRate,
+        accounts: currentAccounts.reduce((acc, currAccount) => {
+          const matchingAccAssets = currAccount.assets.filter(
+            (accAsset) => accAsset.uuid === asset.uuid
+          );
+          if (matchingAccAssets.length) {
+            return [
+              ...acc,
+              ...matchingAccAssets.map((accAsset) => ({
+                address: currAccount.address,
+                ticker: accAsset.ticker,
+                amount: weiToFloat(accAsset.balance, accAsset.decimal),
+                fiatValue: convertToFiatFromAsset(accAsset, exchangeRate),
+                label: currAccount.label
+              }))
+            ];
+          }
+          return acc;
+        }, [] as BalanceAccount[])
+      };
+    })
+    .sort((a, b) => b.fiatValue - a.fiatValue);
+
+  const totalFiatValue = balances.reduce((sum, asset) => {
```

Comment:
Use existing utility methods for this or abstract them out from `StoreProvider`

Reply:
I'm fairly sure we have existing logic for this.

Reply:
just duplicated ones. i extracted this out too c: